packages/website/docs/introduction.md

```diff
@@ -3,6 +3,38 @@ id: introduction
 title: What is Autocomplete?
 ---
 
-import Draft from './partials/draft.md'
+Autocomplete is an open-source, production-ready JavaScript library for building autocomplete experiences.
 
-<Draft />
+A user types into an input, and the autocomplete "completes" their thought by providing full terms or results: this is the very base of an autocomplete experience.
+
+For example, try typing the letter "s" in the search box on the top right of this site. You can directly navigate to documentation on concepts like "**s**ources" and "**s**tate". If you continue typing, say "st", the results update. You see pages having to do with "**st**ate," "getting **st**arted," and "**st**atic sources."
```

Comment:
Displaying search matches with both strong + underline might make it more obvious, wdyt?

![image](https://user-images.githubusercontent.com/6137112/104897977-58c2ac00-5979-11eb-8480-b793f0a42467.png)


```md
<strong><u>s</u></strong>
```

Reply:
https://github.com/algolia/autocomplete.js/pull/393/commits/23b7ad0ef64bd1fad40e57c58c3209868a5f82dc